data/json/furniture_and_terrain/terrain-manufactured.json

```diff
@@ -640,6 +640,7 @@
         { "item": "RAM", "count": [ 4, 8 ] },
         { "item": "cable", "charges": [ 8, 16 ] },
         { "item": "small_lcd_screen", "count": [ 2, 4 ] },
+        { "item": "large_lcd_screen", "count": 1 },
```

Comment:
2..4 small LCD screens AND one large LCD screen, that's right?

Reply:
In most cases I would agree that a device should only have a single screen, but this is a nuclear generator. I imagine it has a large screen and then a few smaller screens used for displaying data and status information. 
However if you disagree just let me know and I will remove the smaller screens.

Reply:
That's OK.